components/script/dom/htmltableelement.rs

```diff
@@ -338,6 +348,22 @@ impl HTMLTableElementMethods for HTMLTableElement {
         Ok(new_row)
     }
 
+    // https://html.spec.whatwg.org/multipage/tables.html#dom-table-deleterow
+    fn DeleteRow(&self, mut index: i32) -> Fallible<()> {
+        let mut rows = self.get_rows();
+        // Step 1.
+        if index == -1 {
+            index = rows.len() as i32 - 1;
+        }
+        // Step 2.
+        if index < 0 || index as usize >= rows.len() {
+            return Err(Error::IndexSize);
+        }
+        // Step 3.
+        rows.sections.remove(index as usize);
```

Comment:
I'm not sure it does remove the row as expected. If someone could confirm please?